tests/migration/test_host_evacuate.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+
+from subprocess import CalledProcessError
+from lib.common import wait_for, wait_for_not
+# The pool needs a shared SR to use `host.evacuate`
+from tests.storage.nfs.conftest import *
+
+# Requirements:
+# From --hosts parameter:
+# - host(A1): first XCP-ng host >= 8.2 with an additional unused disk for the SR.
+# - hostA2: Second member of the pool. Can have any local SR. No need to specify it on CLI.
+# - second_network: A 2nd network of the pool, NOT the management interface, with PIF plugged on all hosts
```

Comment:
Isn't this is from  `--second-network` parameter?

Reply:
Yes why?

Reply:
It's in the "From --hosts parameter" section of the comment but it does not come from the `--hosts` parameter.